CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Changed
+
+- The charge value that's written to the results mzTab file is now an integer value to comply
+with the mzTab standard.
```

Comment:
**todo:** Remove the newline. Also this should probably go under the "Fixed" rather than "Changed" category.

I'd also rephrase a bit more concrete for end users: "Precursor charges are exported as integers instead of floats in the mzTab output file, according to the mzTab specification."

Reply:
Sounds good, I edited the changelog entry.